app/views/spree/admin/gdpr_requests/_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<div data-hook="admin_gdpr_request_form_fields">
+  <div class="row">
+    <div class="left col-12" data-hook="admin_gdpr_request_form">
+      <div class="row">
+        <div class="col-6" data-hook="admin_gdpr_request_form_user">
+          <%= f.field_container :user do %>
+            <%= f.label :user_id, Spree::User.model_name.human %><br />
+            <%= f.collection_select(:user_id, @users, :id, :email, { include_blank: t('spree.match_choices.none') }, { class: 'custom-select fullwidth' }) %>
+          <% end %>
+        </div>
+
+        <div class="col-6" data-hook="admin_gdpr_request_form_intent">
+          <%= f.field_container :intent do %>
+            <%= f.label :intent, Spree::GdprRequest.human_attribute_name(:intent) %><br />
+            <%= f.select :intent, @intents, { include_blank: t('spree.match_choices.none') }, { class: 'custom-select fullwidth' } %>
```

Comment:
If the blank option has a label, users might mistakenly believe the field has been filled. I think it would be best to keep it entirely blank (`include_blank: true`) to avoid confusion.

Reply:
I think that since the field is required, I should remove the `include_black` option, and shows as selected the first intent. WDYT?

Reply:
@vassalloandrea the downside of *not* including a blank option is that the user might forget to select an intent and instead go with the default one, which is not what we want here.